packages/devtools_app/lib/src/timeline/timeline_model.dart

```diff
@@ -1005,12 +1020,7 @@ class AsyncTimelineEvent extends TimelineEvent {
                 firstNewEventParent != null &&
                 lastEventParent == firstNewEventParent &&
                 lastEventAtLevel.time.end >= lastEventParent.time.end;
-        if (eventsOverlap ||
-            newEventStartsBeforeLastEvent ||
-            siblingsWouldCauseOverlappingGuideline) {
-          // [event] does not fit at [displayRow].
-          return false;
-        }
+        if (siblingsWouldCauseOverlappingGuideline) return false;
```

Comment:
nit: maybe remove the siblingsWouldCauseOverlappingGuidlines as it isn't really needed given the the comment describing what it is.

Reply:
done